SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -38,6 +58,7 @@ class SPATIALGDK_API ASpatialDebugger :
 
 public:
 
+	virtual void GetLifetimeReplicatedProps(TArray< FLifetimeProperty > & OutLifetimeProps) const override;
```

Comment:
TArray< FLifetimeProperty > & OutLifetimeProps

Is this the correct formatting? Perhaps:

TArray<FLifetimeProperty>& OutLifetimeProps